tools/vscode-azurewebpubsub/package.json

```diff
@@ -67,6 +74,12 @@
                 "title": "%azureWebPubSub.service.createInPortal%",
                 "category": "Azure Web PubSub",
                 "icon": "$(add)"
+            },
+            {
+                "command": "azureWebPubSub.service.create",
+                "title": "%azureWebPubSub.service.create%",
+                "category": "Azure Web PubSub",
```

Comment:
is it possible/better we separate WebPubSub/SocketIO creation in the commands? e.g. create web pubsub, and create web pubsub for socketio